boilerplate/openshift/golang-codecov/README.md

```diff
@@ -0,0 +1,14 @@
+# Conventions for code coverage for our Golang projects
+
+- [Code coverage](#code-coverage)
+
```

Comment:
This README could mention the `make` targets that are also included, and how this convention ties into the `coverage` CI target.

On that note, should we include a `prow-config` script that lays down just the codecov subset of the [one from golang-osd-operator](https://github.com/openshift/boilerplate/blob/7d81a9d4ba6ed1b17ed0f0ceee85cb9def9884d4/boilerplate/openshift/golang-osd-operator/prow-config)?

Looking to the future, assembling piecemeal conventions like this, each convention's prow-config will have to be more clever, actually parsing the yaml and ensuring its specific section(s) are copacetic rather than laying down the whole file.